docs/conf.py

```diff
@@ -215,18 +215,18 @@ def setup(app):
 # -- Options for LaTeX output ---------------------------------------------
 
 latex_elements = {
-# The paper size ('letterpaper' or 'a4paper').
-#'papersize': 'letterpaper',
+    # The paper size ('letterpaper' or 'a4paper').
+    # 'papersize': 'letterpaper',
 
-# The font size ('10pt', '11pt' or '12pt').
-#'pointsize': '10pt',
+    # The font size ('10pt', '11pt' or '12pt').
+    # 'pointsize': '10pt',
 
-# Additional stuff for the LaTeX preamble.
-#'preamble': '',
+    # Additional stuff for the LaTeX preamble.
+    # 'preamble': '',
 
-# Latex figure (float) alignment
-#'figure_align': 'htbp',
-} # type: Dict[str, str]
+    # Latex figure (float) alignment
+    # 'figure_align': 'htbp',
+}  # type: Dict[str, str]
```

Comment:
We should exclude this file -- it's a third-party template we don't really want to modify.